src/files/BrsFile.ts

```diff
@@ -1287,32 +1296,14 @@ export class BrsFile implements BscFile {
                         nameRange: namespaceStatement.nameExpression.location?.range,
                         lastPartName: part.text,
                         lastPartNameLower: lowerPartName,
-                        functionStatements: new Map(),
                         namespaceStatements: [],
-                        namespaces: new Map(),
```

Comment:
This data was not used at all, and so there were a lot of maps created (and cached) that were not used